caches.json

```diff
@@ -0,0 +1,9 @@
+{
+  "caches": [
+    {
+      "id": "google",
+      "name": "Google AMP Cache",
+      "updatePingApiUrlPrefix": "https://cdn.ampproject.org/"
+    }
+  ]
+}
```

Comment:
What's involved in getting the cloudflare cache listed here?  Where is this mapping used?  Should we create a follow up PR to add the cloudflare cache?